crates/top/rerun_c/src/video.rs

```diff
@@ -0,0 +1,51 @@
+use crate::{CError, CErrorCode, CStringView};
+
+#[allow(unsafe_code)]
+#[no_mangle]
+pub extern "C" fn rr_video_asset_read_frame_timestamps_ns(
```

Comment:
More work than you likely want to tackle today, but once we have "dataframe" APIs in rerun_c, I think this is a good candidate to return the same arrow-based columnar structures we use in our query APIs instead of a bare array like this.

Reply:
agreed on both accounts 😄